Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM alpine:latest
 
-RUN apk -q update ; apk -q add maven git jo
+RUN apk -q update ; apk -q add maven git xq
```

Comment:
why are we building the app inside docker. This should be change and move to build time

Reply:
Not everybody has Java and Maven installed on their systems, so this allows them to build and run the conformance tool.